JOSS/paper.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'rl: A Modular codebase for reinforcement learning models training, testing and visualization.'
+tags:
+  - Python
+  - Reinforcement Learning
+  - Tensorflow
+  - Q-Learning
+authors:
+  - name: Aidan N. Gomez
+    affiliation: 1
+  - name: Bryan M. Li
+    affiliation: 1
+  - name: David Tao
+    affiliation: 1
+  - name: Siddhartha Rao Kamalakara
+    affiliation: 1
+  - name: Siddhartha Rao Kamalakara
```

Comment:
duplicate name. can we follow the order from the current readme?

Reply:
Sure. Let me fix it